astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1756,8 +1757,10 @@ def test_is_busday(self):
 untested_functions |= financial_functions
 
 deprecated_functions = {
-    np.asscalar, np.rank
+    np.asscalar
     }
+if NUMPY_LT_1_18:
```

Comment:
@mhvk , like this?

Reply:
I think yes, given he said to do the same for the test. With this assumption I go ahead and merge this.